holoviews/util/settings.py

```diff
@@ -361,7 +361,7 @@ def update_options(cls, options, items):
     @classmethod
     def initialize(cls, backend_list):
         cls.backend_list = backend_list
-        backend = cls.options.get('backend', Store.current_backend)
+        backend = Store.current_backend
```

Comment:
@philippjfr I don't quite know why I had to make this change. I'll try to summarize:

* In the magic tests, ``%unload_ext`` is called which meant that ``initialize`` was called multiple times because that unsets ``_loaded`` on the notebook extension. If the ``options`` dictionary had a value, this meant state was bleeding across tests and breaking.
* I can't tell what changed: I believe ``initialize`` would have been called multiple times in the tests before this PR too.

That said, why isn't this an appropriate fix? When you initialize, it should reset the state even if initialize was called before. Why use a value from ``cls.options`` when the purpose here is to reset it to the defaults (line below?).